src/chain.h

```diff
@@ -466,7 +466,7 @@ class CDiskBlockIndex : public CBlockIndex
             READWRITE(spentSerials);
 	    }
 
-        if (!(nType & SER_GETHASH)) {
+        if (!(nType & SER_GETHASH) && nHeight >= Params().GetConsensus().nSigmaStartBlock) {
             READWRITE(mintedPubCoinsV3);
```

Comment:
Is it a problem if these maps are empty? I believe they were empty, and must be still serialized without a problem no?

Reply:
serialization support empty map. you can take a look serialization function at line 1056 in file `src/serialize.h`.

Reply:
ok, got it. thanks!